File: src/main/java/com/kn/knwremodel/service/CollegeService.java
package com.kn.knwremodel.service;

import java.util.List;
import org.springframework.stereotype.Service;
import com.kn.knwremodel.repository.CollegeRepository;
import lombok.RequiredArgsConstructor;


@Service
@RequiredArgsConstructor
public class CollegeService {
    private final CollegeRepository CollegeRepo;
    public List<String> findAllMajor() {
        return CollegeRepo.findAllMajor();
    }
}

File: src/main/kotlin/injea/knwremodel/College/CollegeService.kt
package injea.knwremodel.College;

import java.util.List;
import org.springframework.stereotype.Service;

import lombok.RequiredArgsConstructor;


@Service
@RequiredArgsConstructor
public class CollegeService {
    private final CollegeRepository CollegeRepo;
    public List<String> findAllMajor() {
        return CollegeRepo.findAllMajor();
    }
}